ledger/sandbox-on-x/reference.conf

```diff
@@ -382,21 +382,21 @@ ledger {
       index-service {
 
         acs-streams {
-          # Number of contract ids fetched from the index for every round trip when serving ACS calls.
+          # Number of event ids to retrieve in a single query (a page of event ids).
```

Comment:
Not sure if it is relevant here or not, but this text was taken from `CliConfig`